Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -66,8 +66,8 @@
 
 # API. Choose only 1. 
 # localhost set up using Visual Studio to enable local testing.
-#hostURL = 'https://360ApiTrain.gordon.edu/'
-hostURL = 'http://localhost:2550/'
+hostURL = 'https://360ApiTrain.gordon.edu/'
```

Comment:
Thank you for setting this back for the merge, it is good to keep the apartapp-develop branch using the same default URLs as the develop branch

Reply:
You're welcome!